Jenkinsfiles/Jenkinsfile.cbc-run-integration-tests

```diff
@@ -0,0 +1,83 @@
+pipeline {
+  agent {
+    kubernetes {
+      defaultContainer "bb2-cbc-build"
+      yamlFile "Jenkinsfiles/cbc-run-integration-tests.yaml"
+    }
+  }
+
+  environment {
+    DJANGO_USER_ID_SALT = credentials("bb2-integration-tests-bfd-salt")
+    DJANGO_USER_ID_ITERATIONS = credentials("bb2-integration-tests-bfd-iterations")
+    BRANCH = "${params.COMMIT_SHA == "" ? params.BRANCH : params.COMMIT_SHA}"
+    FHIR_URL = "${params.FHIR_URL}"
+    DJANGO_FHIR_CERTSTORE = "./certstore/"
+  }
+
+  parameters {
+    string(
+      name: "BRANCH",
+      defaultValue: "master",
+      description: "The branch of the bluebutton-web-server repo to test."
+    )
+    string(
+      name: 'COMMIT_SHA',
+      defaultValue: "",
+      description: 'The commit sha to send status updates for. If specified, this variable overrides BRANCH.'
+    )
+    string(
+      name: 'FHIR_URL',
+      defaultValue: "https://prod-sbx.bfdcloud.net/v1/fhir/",
+      description: 'The default FHIR URL for the back end BFD service.'
+    )
+  }
+
+  stages {
+    stage("Checkout BRANCH or COMMIT_SHA") {
+      steps {
+        checkout([
+          $class: 'GitSCM',
+          branches: [[
+            name: "${env.BRANCH}"
+          ]],
+          doGenerateSubmoduleConfigurations: false,
+          userRemoteConfigs: [[
+            url: 'https://github.com/CMSgov/bluebutton-web-server.git'
+          ]]
+        ])
+      }
```

Comment:
This works and checks out the env.BRANCH for the container. However, there is this warning message. Is there a better way to do this, or is this OK?
```
Warning: JENKINS-30600: special launcher org.csanchez.jenkins.plugins.kubernetes.pipeline.ContainerExecDecorator$1@XXXXX; decorates RemoteLauncher[hudson.remoting.Channel@XXXXXX:JNLP4-connect connection from ip-XX-XX-XX-XX will be ignored (a typical symptom is the Git executable not being run inside a designated container)
```